Pizza Slice Divider.html

```diff
@@ -4,10 +4,26 @@
     <meta charset="utf-8">
     <title>Pizza Slice Divider</title>
     <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
   </head>
   <body>
-<h1>Pizza Slice Divider</h1>
-<h2>Lets work it out and be fair</h2>
+<div class="inputs">
+  <div>
+<label for="people">How many people?</label>
+<input type="number" name="people" id="people" value="2" min="2" max="100" required>
```

Comment:
Well done setting a `min` value, so that it is not possible to enter invalid numbers (like zero, which would make division difficult, or negative)